src/riak_kv_get_fsm.erl

```diff
@@ -180,23 +180,35 @@ init({test, Args, StateProps}) ->
     {ok, validate, TestStateData, 0}.
 
 %% @private
-prepare(timeout, StateData=#state{bkey=BKey={Bucket,_Key},
+prepare(timeout, StateData=#state{bkey = {Bucket, PossiblyTSKey},
```

Comment:
Instead of doing TS key handling internal to this function, I would like to see two function heads: 

```
prepare(timeout, StateData=#state{bkey={Bucket, Key}}...) when is_binary(Key) ->
    prepare(timeout, {Bucket, {Key, Key}}); %% pseudocode
prepare(timeout, StateData=#state{bkey={Bucket, {PK, LK}}} ->
```